### DISCUSSION TITLE
Vorta: public key for the ppa is missing

### DISCUSSION THREAD

**Discussion by thob** [borgbase/vorta#797] [General]
**public key went missing**
wanted to download Vorta for Ubuntu via PPA but got public key is missing. Couldn't download it from hkp://pool.sks-keyservers.net:80 either

**Comment by samuel-w** [comment]
Are you running `sudo add-apt-repository ppa:samuel-w1/vorta`? I used that and it works in a live iso.

**Comment by samuel-w** [comment]
Also are you on Ubuntu 20.04 or greater?
This is the key: http://keyserver.ubuntu.com/pks/lookup?search=0x814E5D10671804FE2C692BA9B6509CEF9A77BD39&fingerprint=on&op=index

**Comment by thob** [comment]
Hmm, I guess I got a more comprehensive _missing key_ issue. I'll check this first. Thanks

**Comment by samuel-w** [comment]
I'd try removing it `sudo add-apt-repository --remove ppa:samuel-w1/vorta` and then adding it back.
